src/data/minors/atmo-sci.ts

```diff
@@ -0,0 +1,48 @@
+import { CollegeOrMajorRequirement } from '../../requirements/types';
+import { includesWithSingleRequirement } from '../../requirements/checkers';
+import { AdvisorGroup } from '../../tools/advisors/types';
+
+const atmoSciMinorRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Two Core Courses',
+    description: 'Any two of these core courses: EAS 3410, EAS 3420, EAS 4470.',
```

Comment:
```suggestion
    description: 'Any two of the following core courses: EAS 3410, EAS 3420, EAS 4470.',
```
For wording consistency